entity_event/models.py

```diff
@@ -799,6 +834,8 @@ class Subscription(models.Model):
     sub_entity_kind = models.ForeignKey(EntityKind, null=True, related_name='+', default=None)
     only_following = models.BooleanField(default=True)
 
+    objects = SubscriptionManager()
```

Comment:
Can you try out this new thing in Django 1.7 for this? https://docs.djangoproject.com/en/1.8/releases/1.7/#calling-custom-queryset-methods-from-the-manager

I've known about this feature for quite some time, and it looks like it is applicable for this model manager